pkg/client/status_test.go

```diff
@@ -39,8 +39,8 @@ func TestGetStatus(t *testing.T) {
 			setup: func(t *testing.T, transport *httpmock.MockTransport) {
 				transport.RegisterResponder(http.MethodGet, "/api/status/",
 					httpmock.NewStringResponder(http.StatusOK, `{
-						"pngx_version": "2.14.7",
-						"server_os": "Linux-6.8.12-8-pve-x86_64-with-glibc2.36",
+						"pngx_version": "2.18.4",
```

Comment:
I would appreciate it if we could leave the pre-existing data unchanged. Instead you can add the extended data as a new testcase.